ocaml/idl/datamodel.ml

```diff
@@ -1216,6 +1216,8 @@ let _ =
     ~doc:"The update precheck stage failed: the server is of an incorrect version." ();
   error Api_errors.update_precheck_failed_out_of_space ["update"; "available_space"; "required_space "]
     ~doc:"The update precheck stage failed: the server does not have enough space." ();
+  error Api_errors.update_precheck_failed_gpgkey_not_imported ["update"]
+    ~doc:"The update precheck stage failed: the update required gpg key is not imported in the host." ();
```

Comment:
@thomassa, thanks for the suggestion. That makes sense.
Update the message accordingly.
Could you please take a look?

Thanks.